listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -40,3 +40,16 @@ def check_connection(self):
         except redis.exceptions.ConnectionError as e:
             self.log.error("Redis ping didn't work: {}".format(str(e)))
             raise
+
+    def store_latest_listens(self, listens):
+        try:
+            all_listens = self.redis.set('latest_listens', listens)
+        except redis.exceptions.ConnectionError as e:
+            self.log.error("The data can't be set due to: {}".format(str(e)))
+
+    def get_latest_listens(self):
+        """ This module is used to get the datas that has been set in the redis
```

Comment:
this docstring should be more clear.

```suggestion
        """ Returns the latest ten listens that have been cached in the redis listenstore.
        """
```